src/hooks/useFiatText.js

```diff
@@ -24,7 +24,6 @@ export const useFiatText = (props: Props) => {
     cryptoExchangeMultiplier = defaultMultiplier,
     appendFiatCurrencyCode,
     nativeCryptoAmount = cryptoExchangeMultiplier,
-    fiatSymbolSpace,
```

Comment:
Note: I found that the fiat symbol is spaced in 2 places right now: WiredBalanceBox and ControlPanel.

Can't really integrate FiatText into WiredBalanceBox because of the new requirement of a wallet so not bothering to touch it for now.

For ControlPanel, however, it looks like there was a bug introduced in ff7595ab0ce81c75abf58ef4da4b973480cb2d52 where the fiat symbol space was hard-coded into the return value of FiatText no matter what.

Don't really see any other way to handle this besides re-introducing the fiatSymbolSpace variable in the FiatText hook and component.

Reply:
Alternatively, maybe we can re-visit the design and change the fiat symbol spacing to be consistent across the app.
Personally, I think it's weird that in the Wallet List there's crypto symbol spacing but not fiat symbol spacing.